processor/src/main/kotlin/permissions/dispatcher/processor/impl/ProcessorUnits.kt

```diff
@@ -6,13 +6,14 @@ import permissions.dispatcher.processor.impl.java.JavaSupportFragmentProcessorUn
 import permissions.dispatcher.processor.impl.kotlin.KotlinActivityProcessorUnit
 import permissions.dispatcher.processor.impl.kotlin.KotlinNativeFragmentProcessorUnit
 import permissions.dispatcher.processor.impl.kotlin.KotlinSupportFragmentProcessorUnit
+import javax.annotation.processing.Messager
 
-fun javaProcessorUnits() = listOf(
-        JavaActivityProcessorUnit(),
-        JavaSupportFragmentProcessorUnit(),
-        JavaNativeFragmentProcessorUnit())
+fun javaProcessorUnits(messager: Messager) = listOf(
```

Comment:
What is the purpose of having `Messager`? Is it for debugging purpose?

Reply:
Yeah, I propagated it to the `ProcessorUnit` so I could print some debug statements while developing!